scripts/generate_microRTPS_bridge.py

```diff
@@ -354,7 +354,7 @@ def generate_agent(out_dir):
     # the '-typeros2' option in fastrtpsgen.
     # .. note:: This is only available in FastRTPSGen 1.0.4 and above
     gen_ros2_typename = ""
-    if ros2_distro and ros2_distro in ['dashing', 'eloquent', 'foxy', 'galactic', 'rolling'] and fastrtpsgen_version >= version.Version("1.0.4"):
+    if ros2_distro and ros2_distro in ['dashing', 'eloquent', 'foxy', 'galactic', 'humble', 'rolling'] and fastrtpsgen_version >= version.Version("1.0.4"):
```

Comment:
this has to be updated on the PX4-Autopilot repo, which then deploys the (updated) script here.

Reply:
I reverted this changed and opened  a pull request to PX4-Autopilot with this update: https://github.com/PX4/PX4-Autopilot/pull/19989